Gallery/TaylorDiagrams/NCL_taylor_3.py

```diff
@@ -0,0 +1,70 @@
+"""
+NCL_taylor_3.py
+===============
+This script illustrates the following concepts:
+   - Creating a basic Taylor diagram
```

Comment:
I recommend specifically mention that this example uses geocat-viz and then link to the documentation page for the Taylor function. This way we can promote our other packages